### PR TITLE
Feature/34 project profile

### PR DESCRIPTION
**Pull request by niver-copa-jalafund**
[AB#34](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/34)

# Overview

34 Projects | CRUD | Frontend | Project's Profile

### CODE REVIEW

File: src/components/ProjectForm/ProjectDetail.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        <Typography align="justify" color="textSecondary">
+          Description: {project.description}
+        </Typography>
+        <Divider variant="middle" />
+        <Typography align="justify" color="textSecondary">
+          Ref.:{project.textInvitation}
+        </Typography>
+        <Divider variant="middle" />
+        <Typography align="justify" color="primary">
+          Contact Info: {project.contact}
```
Comment (DaleneyRuben) [inline src/components/ProjectForm/ProjectDetail.jsx]: you can use object destructuring here

File: src/components/ProjectForm/ProjectDetail.jsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import axios from "axios";
```
Comment (DaleneyRuben) [inline src/components/ProjectForm/ProjectDetail.jsx]: This doesn't match the design, for example the Publish label where that came from?

Comment (DaleneyRuben) [reply src/components/ProjectForm/ProjectDetail.jsx]: change file name to Details.jsx then the file structure will look like this:
```
resumes/
   ...
projects/
   Details.jsx
```

File: src/components/ProjectForm/ProjectDetail.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+import {
+  Button,
+  Container,
+  Divider,
+  Grid,
+  Typography,
+} from "@material-ui/core";
+import React, { useEffect, useState } from "react";
+
+const Project = () => {
```
Comment (DaleneyRuben) [inline src/components/ProjectForm/ProjectDetail.jsx]: change to this 
```
function Details() {
}
```

**Review by DaleneyRuben — CHANGES_REQUESTED** [review]

File: src/components/ProjectForm/ProjectDetail.jsx (hunk 3 above)
Comment (niver-copa-jalafund) [reply src/components/ProjectForm/ProjectDetail.jsx]: is done

File: src/components/ProjectForm/Detail.jsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      setProject(res.data);
+    }
+    fetchData();
+  }, []);
+  const name = project.name;
+  const logo = project.logo;
+  const creationDate = project.creationDate;
+  const description = project.description;
+  const textInvitation = project.textInvitation;
+  const contact = project.contact;
```
Comment (jorge-lema-jalafund) [inline src/components/ProjectForm/Detail.jsx]: We don't do this . Please remove it.

File: src/components/ProjectForm/Detail.jsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+} from "@material-ui/core";
+import React, { useEffect, useState } from "react";
+
+function Details() {
+  const { id } = useParams();
+  const [project, setProject] = useState([]);
+  useEffect(() => {
+    async function fetchData() {
+      const res = await axios.get(
+        `https://jsonplaceholder.typicode.com/users/${id}`
```
Comment (jorge-lema-jalafund) [inline src/components/ProjectForm/Detail.jsx]: you shouldn't be using axios in your component. Please extract it to another file used for api calls
Also, I heard the backend for projects was ready. You could use it

**Review by jorge-lema-jalafund — CHANGES_REQUESTED** [review]

File: src/components/ProjectForm/Detail.jsx (hunk 5 above)
Comment (niver-copa-jalafund) [reply src/components/ProjectForm/Detail.jsx]: is done

File: src/components/ProjectForm/useFetch.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    fetchData();
+  }, [url]);
+
+  return {
+    data,
+    error,
+  };
+}
+
+export default useFetch;
```
Comment (jorge-lema-jalafund) [inline src/components/ProjectForm/useFetch.js:26]: There's an existing file for this. ProjectsAPI.js .  Create your function inside that one
image

**Review by jorge-lema-jalafund — CHANGES_REQUESTED** [review]

File: src/components/ProjectForm/useFetch.js (hunk 6 above)
Comment (niver-copa-jalafund) [reply src/components/ProjectForm/useFetch.js:26]: That ins´t in the file, 
image

**Pull request closed without merging**